server/routes/api.js

```diff
@@ -2,8 +2,125 @@ const express = require('express');
 
 const router = express.Router();
 
-router.get('/', (req, res) => {
-  res.status(200).json({ message: 'hi' });
-});
+/**
+ * When it is triggered: should be constantly checked on interval by logged in user
+ * What it does: checks for and returns any updates to open parties associated with current user; 
```

Comment:
Maybe we can change the wording to remove 'any updates', just for clarity?

Reply:
sure